lib/src/rules/avoid_print.dart

```diff
@@ -5,6 +5,7 @@
 import 'package:analyzer/dart/ast/ast.dart';
 import 'package:analyzer/dart/ast/visitor.dart';
 import 'package:analyzer/dart/element/element.dart';
+import 'package:linter/src/util/flutter_utils.dart';
```

Comment:
I think this needs to get relativized...

Reply:
Done